com.unity.netcode.gameobjects/Tests/Runtime/Metrics/MessagingMetricsTests.cs

```diff
@@ -20,6 +20,7 @@ public class MessagingMetricsTests : DualClientMetricTestBase
 
         protected override int NbClients => 2;
 
+        [Ignore("Snapshot transition")]
```

Comment:
Hi @becksebenius-unity 
Just  wanted to point out a bunch of tests for metrics are getting disabled, here, until we figure out how Snapshot can trigger Metrics and have those tests passing